sysapps/device_capabilities/cpu_info_provider_android.cc

```diff
@@ -20,6 +20,8 @@ const char kProcLoadavg[] = "/proc/loadavg";
 namespace xwalk {
 namespace sysapps {
 
+void CPUInfoProvider::init() {}
```

Comment:
do we anticipate anything here? maybe just put the stuff to the constructor in the windows-specific class (not introduce init() method)?